src/engine/mailprivacy_smime.c

```diff
@@ -65,6 +65,9 @@
 #include <stdio.h>
 #include <ctype.h>
 #include <libetpan/libetpan-config.h>
+#include <errno.h>
+
+#include "../data-types/wrappers.h"
```

Comment:
On Thu, Mar 05, 2015 at 02:31:37PM -0800, Hoà V. DINH wrote:
> > @@ -65,6 +65,9 @@
> >  #include <stdio.h>
> >  #include <ctype.h>
> >  #include <libetpan/libetpan-config.h>
> > +#include <errno.h>
> > +
> > +#include "../data-types/wrappers.h"
> You should rely here on includes search path instead.

Then it does not work (tested on my machine here). The Makefiles are
pretty complex, what would I have to do to enable the include search
path in these files?

Yours,
VB.
-- 
Volker Birk
Oberer Graben 4, 8400 Winterthur, Schweiz
mailto:vb@dingens.org  http://fdik.org